cypress/e2e/signUp.cy.js

```diff
@@ -1,5 +1,29 @@
+import { generateUser } from '../support/generate';
+
+Cypress.on('uncaught:exception', () => {
+  return false;
+});
+
 describe('Sign Up page', () => {
   it('should provide an ability to register new account', () => {
+    const { email, username, password } = generateUser();
+
+    cy.visit('https://react-redux.realworld.io/#/register?_k=spysa9');
+
+    cy.get('[placeholder=Username]')
+      .type(username);
+
+    cy.get('[placeholder=Email]')
+      .type(email);
+
+    cy.get('[placeholder=Password]')
+      .type(password);
+
+    cy.get('button[type=submit]')
+      .should('exist').click();
```

Comment:
The `.should('exist')` assertion before the click action might not be necessary. Cypress automatically waits for elements to exist before executing actions (like `click`). If the button doesn't exist, the test will fail anyway.